### PR TITLE
Solution [Svystun Bohdan]

### PR DESCRIPTION
**Pull request by BSvystun2707**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -7,4 +7,63 @@ describe('getCoinCombination', () => {
...
+      expect(getCoinCombination(6))
+        .toEqual([1, 1, 0, 0]);
+    });
+
+    it(`should return [2, 1, 0, 0] for 7 cents`, () => {
+      expect(getCoinCombination(7))
+        .toEqual([2, 1, 0, 0]);
+    });
+
+    it('should return [3, 1, 0, 0] for 8 cents', () => {
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:35]: This test isn't needed as per the test design techniques) 
As per boundary values you should cover with inputs 1, 5, 10, 25+/- 1 and 1, 5, 10, 25 * 2 (If by this combination you will see duplicates - remove it)
Also add test which will return [1,1,1,1] as well

**Review by 4ksandrey — CHANGES_REQUESTED** [review]

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -7,4 +7,63 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:7]: I can't see any changes in your PR, revise it and fix according to the comment below

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
Check the comments and don't send task on review without changes, thanks

File: src/getCoinCombination.test.js [hunk 3]
```diff
@@ -7,4 +7,72 @@ describe('getCoinCombination', () => {
...
+
+    it('should return [4, 1, 0, 0] for 10 cents', () => {
+      expect(getCoinCombination(9)).toEqual([4, 1, 0, 0]);
+    });
+
+    it('should return [0, 0, 1, 0] for 10 cents', () => {
+      expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+    });
+
+    it(`should return [4, 0, 2, 0] for 25 cents`, () => {
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:40]: 24?)

File: src/getCoinCombination.test.js [hunk 4]
```diff
@@ -7,4 +7,72 @@ describe('getCoinCombination', () => {
...
+
+    it('should return [4, 0, 0, 0] for 4 cents', () => {
+      expect(getCoinCombination(4)).toEqual([4, 0, 0, 0]);
+    });
+
+    it('should return [0, 1, 0, 0] for 5 cents', () => {
+      expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+    });
+
+    it('should return [4, 1, 0, 0] for 10 cents', () => {
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:32]: for 9 I believe)

File: src/getCoinCombination.test.js [hunk 5]
```diff
@@ -7,4 +7,72 @@ describe('getCoinCombination', () => {
...
+
+    it(`should return [0, 0, 0, 2] for 50 cents`, () => {
+      expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+    });
+
+    it('should return [4, 0, 2, 3] for 99 cents', () => {
+      expect(getCoinCombination(99)).toEqual([4, 0, 2, 3]);
+    });
+
+    it('should return [4, 0, 0, 2] for 102 cents', () => {
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:68]: should return [4, 0, 0, 2] for 102 cents 
but you have tested 100 with output [0,0,0,4] 
I'm a little confused)

File: src/getCoinCombination.test.js (hunk 2 above)
Comment (4ksandrey) [reply src/getCoinCombination.test.js:7]: Ok, good changes, but revise please test description and fix it please, check comments below

File: src/getCoinCombination.test.js [hunk 6]
```diff
@@ -7,4 +7,72 @@ describe('getCoinCombination', () => {
...
+  describe('Currency calculator', () => {
+    it('should return [0, 0, 0, 0] for 0 cents', () => {
+      expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+    });
+
+    it(`should return [1, 0, 0, 0] for 1 cent`, () => {
+      expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+    });
+
+    it(`should return [2, 0, 0, 0] for 1 cent`, () => {
```
Comment (4ksandrey) [inline src/getCoinCombination.test.js:20]: for 2 cents?)

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
Check the comments

**Review by 4ksandrey — APPROVED** [review]
Accepted, well done
